cobalt/browser/BUILD.gn

```diff
@@ -30,6 +30,8 @@ source_set("browser") {
     ":embed_polyfilled_javascript",
     "//cobalt/browser/crash_annotator",
     "//cobalt/browser/crash_annotator/public/mojom",
+    "//cobalt/browser/h5vcc_system",
```

Comment:
@hlwarriner and I talked, and we think this should be renamed system at the mojo/implementation layer. It would still be named h5vcc_system in blink, but everything below that would be renamed to not include the h5vcc_ prefix.

Reply:
talked offline, we'll keep it the same for now